openslides/agenda/models.py

```diff
@@ -274,17 +274,6 @@ class Meta:
     def __str__(self):
         return self.title
 
-    def delete(self, skip_autoupdate=False, *args, **kwargs):
```

Comment:
This method cares that a projector element is disabled if the respective database instance is removed. I think we will still need it. (Also the same for all other apps.)

Reply:
You are right. I don't know what I was thinking.

But I don't like, that we override the delete method of any model with nearly the same code. I try to find another way to solve the problem.

Reply:
This might go in the direction of https://github.com/OpenSlides/OpenSlides/pull/4061

Reply:
I think it is a different problem. The issue #4061 is about autoupdating data if something else changes. So you have a valid database but you have to inform the clients in an efficient way (you could send all data on any change. That would solve #4061).

The problem here is, that the database itself gets corrupted. When an element is deleted, the id stays in the projector config. So a projector (that was valid a moment ago) shows an error `element not found` in the next moment. Maybe this is the correct behavior.

Django solves this for ForeignKeys with the `on_delete` keyword: https://docs.djangoproject.com/en/2.1/ref/models/fields/#django.db.models.ForeignKey.on_delete

Maybe we should decide what result we want before we implement something. The possibilities are:
* remove the projector element from all projectory -> The projector would still show a message or a countdown but not the element anymore.
* show an error, or at least send an error to the client, so the client can decide what to do. The error would be `element not found` and not `element was deleted`.
* Switch to a default projector element set if an element is deleted. So a countdown or a message also disappears. 
* The same as before but also switch to an default projector element set if an element is not found.

I would like the error-message, because it is the easiest to implement and the client is free to make a decision. For example an admin client how receives this message could automatically remove the projector element or shows a pop-up to the admin, that he has to do something.

Reply:
see https://github.com/OpenSlides/OpenSlides/issues/4105